### PR TITLE
Move assignment_candidate_ids into meeting_user

### PR DESCRIPTION
**Pull request by reiterl**
Update model, actions and tests.

### CODE REVIEW

**Review by r-peschke — CHANGES_REQUESTED** [review]

File: openslides_backend/presenter/get_user_related_models.py [hunk 1]
```diff
@@ -151,18 +151,19 @@ def get_meetings_data(self, user_id: int) -> List[Dict[str, Any]]:
...
+                ["speaker_ids", "submitted_motion_ids", "assignment_candidate_ids"],
             )
             speaker_ids = []
             submitter_ids = []
+            candidate_ids = []
             if meeting_users:
                 for meeting_user in meeting_users.values():
                     speaker_ids = meeting_user.get("speaker_ids", [])
                     submitter_ids = meeting_user.get("submitted_motion_ids", [])
+                    candidate_ids = meeting_user.get("assignment_candidate_ids", [])
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/get_user_related_models.py:166]: No, I just want to get the first and only item. But I changed the code, to make it clear.

File: openslides_backend/presenter/get_user_related_models.py [hunk 2]
```diff
@@ -151,18 +151,19 @@ def get_meetings_data(self, user_id: int) -> List[Dict[str, Any]]:
...
             meeting_users = self.datastore.filter(
-                "meeting_user", filter_, ["speaker_ids", "submitted_motion_ids"]
+                "meeting_user",
+                filter_,
+                ["speaker_ids", "submitted_motion_ids", "assignment_candidate_ids"],
             )
             speaker_ids = []
             submitter_ids = []
+            candidate_ids = []
             if meeting_users:
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/presenter/get_user_related_models.py:162]: meeting_users can be an empty dict and I ran into an error if I missed the initialization here.

**Review by r-peschke — APPROVED** [review]
If you want you can merge, but check the comment

File: openslides_backend/action/actions/motion/create_base.py [hunk 3]
```diff
@@ -58,7 +60,28 @@ def create_submitters(self, instance: Dict[str, Any]) -> None:
         self.apply_instance(instance)
         weight = 1
         for user_id in submitter_ids:
-            data = {"motion_id": instance["id"], "user_id": user_id, "weight": weight}
+            meeting_user = self.datastore.filter(
```
[the remark this answers is not in the document]
Comment (reiterl) [reply openslides_backend/action/actions/motion/create_base.py]: I will do that later.

**Pull request merged**